source/aks/upgrading-aks-clusters.html.md.erb

```diff
@@ -51,6 +51,13 @@ If there are deprecations, create a plan to address them before upgrading any cl
 
 If there are no apiVersions being deprecated, then you can proceed with the upgrade, going through the environments above in order.
 
+## Patch Version Consistency
```

Comment:
Provide an example of the different versions based on your experience e..g 1.30.2 and 1.30.0